source/Generic/PlayState/Enums/GameState.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace PlayState.Enums
+{
+    public enum PlayStateGameState
```

Comment:
I think the name of the enum and its values are not very clear in what they do or what they purpose is. Can you change it to something else please? Maybe something like `AutomaticStateSwitchStatus` with `Enabled` and `Disabled` values but if you have a better idea let me know

Reply:
Sure, fine for me.

Reply:
Done.